### DISCUSSION TITLE
Multivariate time series forecasting

### DISCUSSION THREAD

**Discussion by pratikgehlott** [awslabs/gluonts#1466] [Q&A] [answered]
## Description
Basically performing time series forecasting with multiple features.

**Comment by miteshyadav** [comment]
Any update on this?

**Comment by timoschowski** [comment, marked as answer]
Multi-variate forecasting with GluonTS is possible, e.g., here is a model that you can use:
https://github.com/awslabs/gluon-ts/issues/1095

Here are some related issues: 
https://github.com/awslabs/gluon-ts/issues/652
https://github.com/awslabs/gluon-ts/issues/1095

**Reply by tienhoang1094** [reply]
how about prophet ?

**Reply by timoschowski** [reply]
If you have a multi-variate target where you want to capture the depencency structure between the target time series ((that's what I think about when talking about multi-variate forecasting), then I'm not aware of Prophet being able to do this. I may be wrong though.

If you have a univariate target and co-variate, prophet can do that, R Forecast (via ARIMAX) as well and GluonTS too.
